### PR TITLE
clean up default content

### PR DESCRIPTION
**Pull request by ascott1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,98 +1,11 @@
...
-## Getting involved
-
-This section should detail why people should get involved and describe key areas you are
-currently focusing on; e.g., trying to get feedback on features, fixing certain bugs, building
-important pieces, etc.
-
-General instructions on _how_ to contribute should be stated with a link to [CONTRIBUTING](CONTRIBUTING.md).
+# CFPB Front-End Team
 
+**Description**:  A repository for the discussion and publication of the CFPB front-end team standards.
```
Comment (Scotchester) [inline README.md]: Let's remove the superfluous **Description:** here. Other than that, :+1: !

Comment (ascott1) [inline README.md]: I like the description!

Comment (Scotchester) [inline README.md]: Sorry, I wasn't clear. I mean the "Description" label preceding the sentence.

Comment (ascott1) [inline README.md]: Ahh, I GET IT NOW.

**Comment by ascott1** [comment]
Updated to remove **Description:** and squashed into that single commit. Thanks! :boom:

**Comment by Scotchester** [comment]
Thanks :D :ship:

**Comment by Scotchester** [comment]
Follow-up:

1. After squashing the commits, did you have to force push to your fork?
2. Could you please update GitHub's repo description (the bit under the name) with the line you added to the README?

**Comment by ascott1** [comment]
> 1. After squashing the commits, did you have to force push to your fork?

Yes

2. Could you please update GitHub's repo description (the bit under the name) with the line you added to the README?

DONE!

**Comment by Scotchester** [comment]
:cool: Thanks!

**Pull request merged**